### DISCUSSION TITLE
Beta Release?

### DISCUSSION THREAD

**Discussion by crhy** [axx0/Civ2-clone#101] [Ideas]
Seems like the game is pretty playable already.  Even if it's rough around the edges, can we get a beta?  Thanks in advance!

**Comment by axx0** [comment]
It's too buggy and featureless even for a pre-alpha, but I'll try to do some sort of release when we get more stuff running.

**Reply by crhy** [reply]
Well I'm beyond pumped!  Thanks guys!!!!

**Comment by reubene** [comment]
I think it might be worth having a discussion on what features we would need for an Alpha release. 

Off the top of my head this would be:
1. Save & Load support
2. A basic AI
3. Completed city screen
4. Working menus

Can you think of anything else?

**Reply by axx0** [reply]
I agree. Also get basic unit commands working (many of which already work).
And making sure the gameplay loop is stable, basically with as few crashes as possible.

**Reply by crhy** [reply]
The idea of a fully open source civ 2 clone on android......  No one will ever work again.  LOL

**Comment by crhy** [comment]
Hey guys!  I'm working on a new Linux distro that is going to be sweet.  It's loosely based on Peppermint Devuan, but with a focus on being easy to use and incorporating fresh flatpaks.  Can't wait for a beta so I can include Civ 2 out of the box.  <3

**Comment by crhy** [comment]
Saw some code updates today!  Yay!  <3

**Comment by crhy** [comment]
Any updates on what you're working on?  I come check all the time.  LOL

**Reply by reubene** [reply]
I'm working on Save/Load but it is slow as I don't have much free time to work on it right now

**Reply by crhy** [reply]
Well thanks for responding anyway!  <3

**Comment by crhy** [comment]
Wow!  It's been a year.  How's everybody doing?  Any low-hanging fruit to attack?

**Comment by crhy** [comment]
So I've included the community texture replacements from this thread in my fork:  https://forums.civfanatics.com/threads/better-terrain-graphics-for-mge-from-eivinds-colonialism-scenario.667221/post-16815095

https://github.com/crhy/AxCiv-FOSS-Assets/tree/master

At this point I think we have most of the art to do a stand alone game without original assets.  What assets are we still missing?

Great work everyone!
